packages/common/styles/src/lib/notional-theme/notional-theme.ts

```diff
@@ -466,14 +466,14 @@ const getThemeData = (themeVariant: PaletteMode) => {
         marginBottom: pxToMargin(12),
       },
       caption: {
-        fontSize: pxToRem(10),
+        fontSize: pxToRem(12),
```

Comment:
This is going to have an effect on a lot of stuff, are we sure want to change it?